modules/core/src/lib/layer.js

```diff
@@ -268,7 +270,7 @@ export default class Layer extends Component {
   // Returns the picking color that doesn't match any subfeature
   // Use if some graphics do not belong to any pickable subfeature
   encodePickingColor(i) {
-    assert((((i + 1) >> 24) & 255) === 0, 'index out of picking color range');
+    assert(i < 16777215, 'index out of picking color range');
     return [(i + 1) & 255, ((i + 1) >> 8) & 255, (((i + 1) >> 8) >> 8) & 255];
```

Comment:
Not part of your updates, but since you're optimizing, you might get a bit of a win by pre-allocating the 3-element array, pulling out the increment, and collapsing the shifts. E.g. something like:
```
let idx = i + 1;
pickColor[0] = idx & 255;
pickColor[1] = (idx >> 8) & 255;
pickColor[2] = (idx >> 16) & 255;
return pickColor;
```

Reply:
Good point. Will do.